Added announcement about GraphQL and Season of Doc (#37)
graphql/foundation.graphql.org: content/news/season-of-docs.md

author: Obinna Ekwuno 
summary: As GraphQL joins the Google Season of Docs program, here's how you can be involved. 
event: Google Season of Docs.
extra:
  title: GraphQL joins Google Season of Doc
  content: The GraphQL Foundation, joins the Google Season of Docs program to bring to incorporate the best part of learning, mentorship, and collaboration on a large scale.
---

# **GraphQL joins Google Season of Docs.** 

Open source communities are one of the best ways to collaborate with a larger scale of developers, it incorporates the best part of learning, mentorship, and collaboration on a large scale. 
**Open-source** software has been proven to create large effective ecosystems of users that improve quality. It has been known for many years that the more a piece of software is used, the quicker bugs **will** be identified. In the **open-source** community when a bug is found, users contribute fixes.
**GraphQL** is an **open**-**source** data query and manipulation language for APIs, and a runtime for fulfilling queries with existing data. **GraphQL** was developed internally by Facebook in 2012 before being publicly released in 2015. 

# **What is Google Season of Docs?** 

> The Google Season of docs is the intersection of technical writing, open-source contribution, and mentorship. In this three month program, you will be working closely with maintainers of open source projects to improve on existing documentation and processes to better the community. 

Reviewer: ```suggestion
The Google Season of Docs is the intersection of technical writing, open-source contribution, and mentorship. In this three month program, you will be working closely with maintainers of open source projects to improve on existing documentation and processes to better the community. 
```

Signed-off-by: Brian Warner <brian@bdwarner.com>

For technical writers who are new to open source, the program provides an opportunity to gain experience in contributing to open source projects. 

For technical writers who're already working in open source, the program provides a potentially new way of working together. Season of Docs also gives open source projects an opportunity to engage more of the technical writing community.

## **How can I be a part of this?** 

Open source organizations apply to be mentor organizations in the Season of Docs which includes a list of documentation project ideas with their applications. Technical writers can look through the list of proposals here are the [project ideas for the GraphQL organization](https://github.com/graphql/foundation/tree/master/mentorship/2020/gsod). 

> When you have selected a project you would like to work on contact Ivan Goncharov at [info@graphql.org](mailto:info@graphql.org) to discuss these ideas after which you will write up a project proposal and submits it to Season of Docs. 

Reviewer: ```suggestion
When you have selected a project you would like to work on, write up a project proposal and submit it to [Season of Docs](https://developers.google.com/season-of-docs). You may continue to refine your proposals until July 9th. If you need clarifications, please contact Ivan Goncharov and Uri Goldshtein by sending a message to [mentorship@lists.foundation.graphql.org](mailto:mentorship@lists.foundation.graphql.org).
```
This might be a better alias for them to mail into. I also rearranged the steps, since prospective applicants without questions can go straight ahead and apply.
Signed-off-by: Brian Warner <brian@bdwarner.com>

Author: Oh thanks, Brian. I will commit these changes.

> The GraphQL organization will reach out to you after they have evaluated the proposals. The accepted technical writers spend a few months working with mentors from the open-source organizations to complete their projects at the end of the program, the Google program administrators announce the projects that finished successfully.

Reviewer: ```suggestion
The GraphQL organization will reach out to you after they have evaluated the proposals. The accepted technical writers spend a few months working with mentors from the open-source organizations to complete their projects. At the end of the program, the Google program administrators announce the projects that finished successfully.
```
Signed-off-by: Brian Warner <brian@bdwarner.com>

Check out the [official technical writer guide.](https://developers.google.com/season-of-docs/docs/tech-writer-guide) 

## **How long is this program?**

> The technical writing applications started on June 9th, 2020 and will last till July 9th, 2020. The selection process for the project submitted to organizations lasts till July 31th 2020. Then Doc development officially begins September 14, 2020, and ends. For a more detailed timeline check out the [official season of docs timeline](https://developers.google.com/season-of-docs/docs/timeline).

Reviewer: ```suggestion
The technical writing applications started on June 9th, 2020 and will last till July 9th, 2020. The selection process for the proposals submitted to organizations lasts till July 31st, 2020. Then Doc development officially begins September 14th, 2020, and ends December 5th, 2020. For a more detailed timeline check out the [official season of docs timeline](https://developers.google.com/season-of-docs/docs/timeline).
```
Signed-off-by: Brian Warner <brian@bdwarner.com>

> _– **Obinna Ekwuno**, Software Enginner,DevRel GatsbyJS_

Reviewer: ```suggestion
_– **Obinna Ekwuno**, Software Engineer, DevRel GatsbyJS_
```
Signed-off-by: Brian Warner <brian@bdwarner.com>
